packages/web-app-files/src/components/AppBar.vue

```diff
@@ -147,14 +145,14 @@ export default {
     ]),
     ...mapState(['route']),
 
-    _cannotCreateDialogText() {
+    newButtonDescription() {
       if (!this.canUpload) {
         return this.$gettext('You have no permission to upload!')
       }
       if (!this.hasFreeSpace) {
         return this.$gettext('You have not enough space left to upload!')
       }
-      return null
+      return "Add files or folders"
```

Comment:
This now adds a tooltip for a non-error case as well. IMO not good. Looks like it's required to separate into two different computed props, one for the tooltip and one for the aria-label.

Reply:
Please check the solution I went for and let me know if you like it!